runtime/service/src/test/java/org/apache/polaris/service/quarkus/admin/PolarisAdminServiceAuthzTest.java

```diff
@@ -151,7 +152,8 @@ public void testCreateCatalogSufficientPrivileges() {
 
   @Test
   public void testCreateCatalogInsufficientPrivileges() {
-    final CatalogEntity newCatalog = new CatalogEntity.Builder().setName("new_catalog").build();
+    final CatalogEntity newCatalog =
+        new CatalogEntity.Builder().setName("new_catalog").setCatalogType("INTERNAL").build();
```

Comment:
Why is this change required?

Reply:
I think the fix is simple, I can create a separate PR to fix it. 

https://github.com/apache/polaris/blob/9cb71f8125d79e9bb16117c9110211394173b821/polaris-core/src/main/java/org/apache/polaris/core/entity/CatalogEntity.java#L109

The builder checks if the `catalogType` is `INTERNAL`, if not, it assumes that the type is `EXTERNAL`, we can flip the condition to check if the type is set to `EXTEERNAL`.  I also think that the change only affects tests, because in other cases, it's generated from polaris-management-service.yml which defaults to `INTERNAL`.

Reply:
Opening a PR for that would be great. Thx!

Let's review the impact of that change on that specific PR (I hope it's ok).

Reply:
Done.

Reply:
Since I opened #1998, I reverted this and other similar changes. The build still passes but I think we should try to get 1998 merged before this change.

Reply:
Thx! Let's merge #1998 first and rebase.

Reply:
Done.